### PR TITLE
Assessment Updates #229

### PR DESCRIPTION
**Pull request by hannahtwobulls**
[Trello](https://trello.com/c/4EUQW8q6/229-as-a-user-i-want-to-input-my-information-in-an-updated-assessment)
[Live Env](https://nj-career-network-dev5.firebaseapp.com/)

• Addition of 'Goal' to initial assessment
• Addition of Month dropdowns
• New Textarea input support
• Pull initial Profile data from assessment
• Adds new Datepicker component for selecting Month/Year

### CODE REVIEW

File: components/profile/Profile.js [hunk 1]
```diff
@@ -19,9 +19,42 @@ function Profile() {
         <Typography component="h1" variant="h2" gutterBottom>
           Profile
         </Typography>
+        <div>{JSON.stringify(profileData)}</div>
```
Comment (hannahtwobulls) [inline components/profile/Profile.js:22]: Happy to remove the work related to initializing the profile data -- I completely expanded the scope of the ticket

File: next.config.js [hunk 2]
```diff
@@ -131,7 +131,7 @@ const appEnvironments = {
     name: 'DEV5',
     showName: true,
     airtable: {
-      apiBase: prodAirtableApiBase,
+      apiBase: liveAirtableApiBase,
```
Comment (hannahtwobulls) [inline next.config.js:134]: Not sure how to go about updating the production airtable

Comment (rachelchenrc) [reply next.config.js:134]: try the airtable dump: npm run airtable:dump

File: components/profile/Profile.js (hunk 1 above)
Comment (rachelchenrc) [reply components/profile/Profile.js]: thanks for the great work! I think it's useful to keep it in this ticket

**Pull request merged**